test/e2e/setup.go

```diff
@@ -49,13 +50,17 @@ import (
 	utillog "github.com/Azure/ARO-RP/pkg/util/log"
 	"github.com/Azure/ARO-RP/pkg/util/uuid"
 	"github.com/Azure/ARO-RP/pkg/util/version"
+	"github.com/Azure/ARO-RP/test/util/dynamic"
 	"github.com/Azure/ARO-RP/test/util/kubeadminkubeconfig"
 )
 
 const (
 	smoke = "smoke"
 )
 
+//go:embed static_resources
```

Comment:
TIL about embedded stuff in go. I found this useful btw: https://www.scaler.com/topics/golang/golang-embed/

Reply:
It's used in guardrail as well.

https://github.com/Azure/ARO-RP/blob/40e7987ef8d88112e081a48e288f1dec41c627ec/pkg/operator/controllers/guardrails/guardrails_config.go#L71-L78